contracts/partial/yToken/lendingMethods.ligo

```diff
@@ -504,7 +481,7 @@ function liquidate(
           then failwith("yToken/borrower-cannot-be-liquidator")
           else skip;
 
-          const maxBorrowInCU : nat = calcCollateralValueInCU(
+          const liquidateCollateral : nat = calcCollateralValueInCU(
```

Comment:
This is good variable name.
This function should have be renamed to calcLiquidateCollateral...